### DISCUSSION TITLE
Failed to parse skin

### DISCUSSION THREAD

**Discussion by flopyssdareal** [umnik1/yaamp#14] [Q&A] [answered]
Когда я впервые зашёл в yaamp, Я поставил свой аккаунт, и открылся плеер. Когда я захотел поменять свой скин через сайт, который указан на сайте yaamp, то когда я его загрузил, и попытался поставить в плеер, он мне пишет: "Failed to parse skin", теперь не могу зайти в плеер. 10 раз переустанавливал, ничего не помогает. Что мне делать? 
When I first logged into yaamp, I set up my account and the player opened. When I wanted to change my skin through the site that is listed on the yaamp website, then when I downloaded it and tried to put it in the player, he writes to me: "Failed to parse skin", now i can not get in the player. reinstalled 10 times, nothing helps. What i need to do?

**Comment by Zzoomrus** [comment, marked as answer]
[[Вот тут](https://github.com/umnik1/yaamp/issues/17)] описано решение проблемы
